editions/webdav/tiddlers/WebDAV sync adaptor plugin.tid

```diff
@@ -0,0 +1,15 @@
+title: WebDAV sync adaptor plugin
+created: 201311152153
+modified: 201805170313
+
+! Features
+
+* Loads skinny tiddlers from entire shared directory at startup/login
+* Subsequently syncs changes back to the server
+* Polls for changes from the server
+
+! Issues
+
+* ''[[$:/DefaultTiddlers]]'' doesn't work because thanks to lazy loading it is only loaded after it is needed
+* Should probably disable PUT saver
+* Ignores ''if-match'' header, so doesn't detect clashes on save
```

Comment:
ouch ... We shold use `if-unmodified-since` header instead. .. I did some experiments replacing the `if-match` stuff with the PUT save and it works great.

Reply:
+1    but it's additional to what tiddlyweb had. Currently I'm just trying to get it to work! :-)

Reply:
I see.